src/main/java/com/google/devtools/build/remote/client/LogParserUtils.java

```diff
@@ -258,9 +306,16 @@ private void printEntriesGroupedByAction(OutputStream outStream)
   /** Print log entries to standard output according to the command line arguments given. */
   public void printLog(PrintLogCommand options) throws IOException {
     try {
-      if (options.groupByAction) {
+      if (options.groupByAction && options.formatJson){
+        System.err.println("You can't use groupByAction with formatJson");
```

Comment:
If an option is specified, don't turn the option combination into an error. Handle the formatJson case first, and ignore the groupByAction.